views/results.erb

```diff
@@ -0,0 +1,44 @@
+<style>
+  body {
+    text-align: center;
+  }
+</style>
+
+<div id="body">
+  <div id="heading">
+    <h1 style="font-family: helvetica">CHITTER</h1>
+  </div>
+  <div id="search heading">
+    <div id="search feedback">
+      <% if @search.length == 1 %>
+      <h3 style="font-family: helvetica">Your search for <%= @keyword %> returned 1 result.</h3>
```

Comment:
I like how you've provided context on the search page for the term they searched for than just the results directly.